packages/cli/mocks/server/handlers.ts

```diff
@@ -1,10 +1,13 @@
 import { http, HttpResponse } from "msw";
 
 export const connectorsListHandler
-= http.get("https://raw.githubusercontent.com/wrtnlabs/connectors/refs/heads/main/connectors-list.json", () => {
+= http.get("https://raw.githubusercontent.com/wrtnlabs/connectors/refs/heads/fix/connector_list/connectors-list.json", () => {
```

Comment:
so when we mock the response, we don't need to change it.
It is possible to forget to change this URL. Inadvertent changes should be avoided.
Since we have a response for the mock, we should use it.

Reply:
refs: cli/mocks, https://mswjs.io/